config/labels/analyzers/clang-tidy.json

```diff
@@ -5272,16 +5324,22 @@
     ],
     "clang-diagnostic-uninitialized": [
       "doc_url:https://clang.llvm.org/docs/DiagnosticsReference.html#wuninitialized",
+      "guideline:sei-cert",
       "profile:default",
       "profile:extreme",
+      "profile:security",
       "profile:sensitive",
-      "severity:MEDIUM"
+      "sei-cert:exp33-c",
+      "severity:HIGH"
```

Comment:
Ditto.

Reply:
uninitialized variable should be high severity.